README.md

```diff
@@ -33,19 +33,37 @@ module.exports = {
     filename: 'bundle.js',
   },
   plugins: [new LazyCompilePlugin()],
-};
+}
 ```
 
 ## Options
 
-|                    Name                     |            Type             | Default | Description                                        |
-| :-----------------------------------------: | :-------------------------: | :-----: | :------------------------------------------------- |
-|              **[`refreshAfterCompile`](#refreshAfterCompile)**              |    `boolean`    | `false`  | Enable/Disable *page refresh* when compilation is finish                    |
-
+|                       Name                        |         Type          |   Default   | Description                                                  |
+| :-----------------------------------------------: | :-------------------: | :---------: | :----------------------------------------------------------- |
+| **[`refreshAfterCompile`](#refreshAfterCompile)** |       `boolean`       |   `false`   | Enable/Disable _page refresh_ when compilation is finish     |
+|             **[`ignores`](#ignores)**             | `string[] | RegExp[]` | `undefined` | Match webpack entry request to be ignored from lazy compiler |
```

Comment:
Type should be `Function[] | RegExp[]`. 

`Match webpack entry request to be ignored from lazy compiler` > `Request to be ignored from lazy compiler.`